src/presence_analyzer/tests.py

```diff
@@ -195,6 +195,7 @@ def setUp(self):
         """
         Before each test, set up a environment.
         """
+        reload(utils)
```

Comment:
Can we have a short comment that this is cache-cleaning? It's not obvious on 1st look so if we have other devs in the project they wouldn't know ;)